### PR TITLE
4.9

### PR DESCRIPTION
**Pull request by pdion891**
Please confirm it make sense to add a new package repo for the MySQL driver.

### CODE REVIEW

**Comment by swill** [comment]
LGTM

**Comment by swill** [comment]
Doesn't look like the `api_changes` made it into this PR for some reason...

**Comment by swill** [comment]
Nevermind.  I missed it because it was compressed `833 additions, 75 deletions not shown because the diff is too large. Please use a local Git client to view these changes.`

File: source/upgrade/_mysql_connector.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
+Install new MySQL connector
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Apache CloudStack |version| require an upgrade of the MySQL connector on CentOS.
```
Comment (karuturi) [inline source/upgrade/_mysql_connector.rst]: for debian, we should document to install python-mysql.connector

**Pull request closed without merging**